### PR TITLE
Can you read Code article, and the FreeCode camp tutorial tidbits

### PR DESCRIPTION
**Pull request by PhilipOakley**
Links to light reading and a quite watchable tutorial with more
advanced topics than usual.

Signed-off-by: Philip Oakley <philipoakley@iee.email>

### CODE REVIEW

**Comment by PhilipOakley** [comment]
Hope it's not too late for the April edition. 
It could always act as a starter for May's edition!

File: rev_news/drafts/edition-86.md [hunk 1]
```diff
@@ -266,6 +267,8 @@ __Git tools and sites__
   Free for, among others, personal use and educational institutions.
   If only it's marketing was less <abbr title="Fear, uncertainty, and doubt">FUD</abbr>-ish
   _("Git [...] can easily destroy days of work in just seconds and requires thousands of keyboard commands and parameters")_...
++ [Git for Professionals – Free Version Control Course])https://www.freecodecamp.org/news/git-for-professionals/)
```
Comment (sivaraam) [inline rev_news/drafts/edition-86.md:270]: I suppose the following change is needed for the link to render properly:

```suggestion
+ [Git for Professionals – Free Version Control Course](https://www.freecodecamp.org/news/git-for-professionals/)
```

**Review by sivaraam — COMMENTED** [review]
Thanks for the PR! I just noticed a minor things that needs to be corrected Markdown syntax wise.

**Comment by chriscool** [comment]
Merged, thanks @PhilipOakley and @sivaraam !

**Pull request merged**